pom.xml

```diff
@@ -131,6 +131,11 @@
             <dependency>
                 <groupId>net.minidev</groupId>
                 <artifactId>json-smart</artifactId>
+                <version>2.4.5</version>
+            </dependency>
+            <dependency>
+                <groupId>net.minidev</groupId>
+                <artifactId>accessors-smart</artifactId>
```

Comment:
Why this is necessary?

Reply:
`json-smart-2.4.5` introduces `accessors-smart-2.4.3` that is not available in Maven Central. Had to have `accessors-smart` remains `2.4.2`. 
```
Failure to find net.minidev:accessors-smart:jar:2.4.3 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
```
Failing CI: https://github.com/opensearch-project/security/runs/2906736355?check_suite_focus=true

Reply:
https://mvnrepository.com/artifact/net.minidev/accessors-smart/2.4.3

Reply:
`accessors-smart-2.4.3.jar` is missing at [here](https://repo.maven.apache.org/maven2/net/minidev/accessors-smart/2.4.3/). But `accessors-smart-2.4.2.jar` is [available](https://repo.maven.apache.org/maven2/net/minidev/accessors-smart/2.4.2/).

Reply:
Created an issue: https://github.com/netplex/json-smart-v2/issues/76